tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -877,56 +879,111 @@ static void setLocationInfoForClangNode(ClangNode ClangNode,
   }
 }
 
+static void setLocationInfoForRange(SourceManager &SM, SourceRange R,
+                                    unsigned BufID,
+                                    LocationInfo &Location,
+                                    bool presumed = false) {
```

Comment:
There's here and `MappedDecl` where the parameter name switches casing, would be nice to have them be consistent one way or the other

Reply:
I fixed this up, but the `MappedDecl` parameter already lives alongside mixed case names: https://github.com/swiftlang/swift/pull/82006/files#diff-d29df7eb941cda5dc4dc36df1f3220fe4468a9b58d4fa8ca251c76337f353589L8672

So maybe I should leave that as is?